### ISSUE TITLE
Allow for non-op users to use the commands

### ISSUE THREAD

**Issue by wildex999** [wildex999/TickDynamic#11]
Add an config entry where you can define users that are allowed to use the commands.
Good for servers where moderators do not have OP permissions.

**Comment by spannerman79** [comment]
You could use permission node system.  I see you have some dev releases for 1.8 minecraft - did you take a look at SpongeAPI yet?

**Comment by wildex999** [comment]
Yeah, I have been watching it on and off. They seem to be nearing release, so I guess I should try to make it compatible with SpongeForge too.

I'm not yet sure how I will be doing permissions. Originally I just figured I'd allow you to add moderators in the config or something. I don't wan't to depend on an external permission system, especially as I try to make TickDynamic compatible with multiple servers, some of which might not support the same permission systems.

**Comment by spannerman79** [comment]
Adding via UUID would be best I think.  That way it wouldn't matter if a user changed their minecraft name it will still be in effect.

Adding or removing the user could be defined by UUID or even the current minecraft name.  The same way that when you add a user to ops or to a whitelist - their UUID is added, the display name is more for ease of recognition when you edit/view said file.

Could also make it so that you could only add that user via console or via direct editing said config file.  As I have (basically) a kitchen sink modpack its easier for me to ssh into the server (or for others to use their supplied multicraft etc web panel if they have hosted minecraft) and exec the command there.
